en/guide/mavlink_version.md

```diff
@@ -33,7 +33,30 @@ sequenceDiagram;
 
 {% endmermaid %}
 
-## Semi-transparent Legacy Radios
+### Semi-transparent Legacy Radios
 
 Some popular legacy radios (e.g. the SiK radio series) operate in semi-transparent mode by injecting [RADIO_STATUS](../messages/common.md#RADIO_STATUS) messages into the MAVLink message stream. Per MAVLink spec these should actually emit a heartbeat with the same system ID and a different component ID than the autopilot to be discoverable. However, an additional heartbeat could be an issue for deployed systems. Therefore these radios can alternatively confirm their *MAVLink 2* compliance by emitting `RADIO_STATUS` in v2 message format after receiving the first v2 MAVLink frame.
 
+
+## Negotiating Versions {#negotiating_versions}
```

Comment:
SO I added this section from the guide. I think these go together - the first part explains how you can verify whether MAVLink 2 is supported or not, and the second is best practice guidance for systems to work out which version should be used for initial and subsequent comms. I think it is still useful?